src/components/fields/checkbox-group/checkbox-group.tsx

```diff
@@ -63,18 +63,23 @@ export const CheckboxGroup = (props: IGenericFieldProps<ICheckboxGroupSchema>) =
 	// =============================================================================
 	// EVENT HANDLERS
 	// =============================================================================
-	const handleChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
-		const value = event.target.value;
-		let updatedStateValues = [...stateValue];
+	const handleChange = useCallback(
```

Comment:
why is `handleChange` memoised? was there a performance penalty?
if so, then the `handleChange` in RadioButtonGroup in https://github.com/LifeSG/web-frontend-engine/pull/58 should be memoised?

similarly `isCheckboxChecked()` is not memoised while `isRadioButtonChecked()` is memoised in the other PR.

Reply:
Actually was thinking need to use useCallback to access correct state values when rendering, but it works without memoised function. So removed memoised implementation.